src/sidebar/components/annotation.js

```diff
@@ -122,18 +122,19 @@ function Annotation({
         />
       )}
 
+      {isEditing && (
```

Comment:
Adjusting rhythm helped me see that these controls don't belong in the footer anyway.